home/templates/base_main.html

```diff
@@ -182,8 +182,7 @@
 
             {% comment %} TODO rewrite star generation so this isn't necessary {% endcomment %}
             {% if request.session.color_scheme == "dark" %}
-                <!-- make sure we're above bootstrap's navbar -->
-                <canvas id="navbar-canvas" style="z-index: 1100"></canvas>
+                <canvas id="navbar-canvas" style="z-index: 1000"></canvas>
```

Comment:
pretty sure no z-index is necessary here at all now that the navbar is transparent.